packages/flutter/lib/src/material/popup_menu_theme.dart

```diff
@@ -68,17 +97,32 @@ class PopupMenuThemeData with Diagnosticable {
   /// new values.
   PopupMenuThemeData copyWith({
     Color? color,
+    Color? surfaceTintColor,
+    Color? shadowColor,
```

Comment:
Small nit, but can we have these under elevation in this order:

elevation,
shadowColor,
surfaceTintColor,

That will match other widgets that support elevation parameters and will make it easier to spot when something is missing. Thx.